app/inputs/transaction_chosen_input.rb

```diff
@@ -5,10 +5,10 @@ def input
     options[:label_method] ||= :name
     options[:value_method] ||= :id
 
-    search_fields[attribute_name] = [collection_items.first.send(options[:value_method].to_sym)] if collection_items.size == 1
+    search_fields[attribute_name] = [collection_items.first.send(options[:value_method].to_sym)] if collection_items.to_a.size == 1
 
     select_options = {:multiple => true, :"data-placeholder" => placeholder_label }
-    select_options.merge!({:disabled => :disabled}) unless collection_items && collection_items.size > 1
+    select_options.merge!({:disabled => :disabled}) unless collection_items.to_a.size > 1
```

Comment:
Even if there's no performance benefit, I'd like to see it extracted if only to comment on why it converts to `Array`; I could easily see myself forgetting in the future why it's there and removing that `#to_a` while refactoring.